Automattic/simplenote-macos
# Updating Last Window Behavior

### Details:
In this PR we're restoring the "Dismissing Last Window won't close Simplenote" behavior.

### Testing:
1. Launch Simplenote
2. Click the `x` at the top left corner of the window
3. In the system menu, click over `Note` > `New`
4. Verify that the app is back in the foreground
5. Close Simplenote again
6. Click over the Simplenote icon in the Dock
7. Verify that we're back onscreen!

@roundhill may i bug you with a quick review?
Thanks in advance!!!

cc @drw158 apologies about the late fix, it's been a long road!!
